netutil/netutil.go

```diff
@@ -0,0 +1,49 @@
+package netutil
+
+import (
+	"net"
+	"strings"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+var (
+	getInterfaceAddrs = (*net.Interface).Addrs
+)
```

Comment:
Ic, I have missed the access to the `.Name` property.
(one of my long-time pet-peeves with Go is that you can't define a property as part of an interface)

In that case I agree that this solution is easier, because otherwise we'd have to wrap `net.Interface` in something which allows us to return the name via a function call, just to make the mocking work. that gets unnecessarily complicated...